funnel/templates/macros.html.jinja2

```diff
@@ -59,24 +59,24 @@
   </div>
 {% endmacro %}
 
-{% macro saveprojectform(project, formid='save-form', txt=false) %}
+{% macro saveprojectform(project, formid='save-form') %}
   <form id={{formid}} class="js-save-form" action="{{ project.url_for('save') }}" method="POST">
     {{ current_view.SavedProjectForm().hidden_tag() }}
-    <p class="no-jshidden zero-bottom-margin">
+    <div class="no-jshidden zero-bottom-margin">
       {%- if current_auth.is_anonymous %}
-        <a class="mui--text-light link-icon"  href="{{ url_for('login') }}" data-ga="Login to save project">{{ faicon(icon='bookmark', baseline=true, css_class='mui--text-light') }}{% trans %}Login{% endtrans %} <span class="mui--hidden-xs mui--hidden-sm">{% trans %}to save{% endtrans %}</span></a>
+        <a class="mui--text-light"  href="{{ url_for('login') }}" data-ga="Login to save project">{{ faicon(icon='bookmark', icon_size='title', baseline=true, css_class='mui--text-light') }}</a>
```

Comment:
I think this needs `{{ url_for('login', next=request.path) }}` to return the user here.